sources/packages/backend/apps/db-migrations/src/migrations/1688501112255-UpdateRegulatoryBodyCode.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class UpdateRegulatoryBodyCode1688501112255
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData(
+        "Update-regulatory-body-ita-code.sql",
+        "EducationPrograms",
```

Comment:
just wondering, since the migration affects 2 different entities. how about creating 2 `.sql` files in the respective entity folders, and add in the same migration file? something like this, 
![image](https://github.com/bcgov/SIMS/assets/77353155/f90d7526-85c2-4617-9dab-cb55a0a7f22c)

Reply:
For history, I think that it would be better as @ann-aot mentioned.
The idea to have the SQL files more table centered is to check the changes along the time with a particular entity, so it would make sense.